src/pushsource/_impl/model/base.py

```diff
@@ -42,6 +43,9 @@ class KojiBuildInfo(object):
     Example: in "kf5-kio-5.83.0-2.el8.next", the release is "2.el8.next".
     """
 
+    id = attr.ib(type=str, default=None, validator=optional_str, converter=int2str)
```

Comment:
May I ask what's the thinking behind storing this as a `str`?  Isn't it always an integer?

Reply:
Changed to `int`